test/core/linter_test.py

```diff
@@ -926,10 +926,6 @@ def test_advanced_api_methods():
     linter = Linter()
     parsed = linter.parse_string(sql)
 
-    # CTEDefinitionSegment.get_identifier
```

Comment:
Removing this from the test, since `get_identifier()` is no longer a segment method.